src/main/java/frc/robot/Robot.java

```diff
@@ -94,5 +97,6 @@ private void setDefaultCommands()
     drivetrain.setDefaultCommand(new Drive(this));
     conveyor.setDefaultCommand(new ConveyorStop(this));
     shooter.setDefaultCommand(new ShooterStop(this));
+    sensors.setDefaultCommand(new UpdateAccel(this));
```

Comment:
Can avoid this by overriding `periodic()` in drivetrain.  The exception would be if you need to use the accel and gyro in subsystems other than the drivetrain would require a separate subsystem.